src/plugins/template/src/compiled_model.cpp

```diff
@@ -143,7 +143,8 @@ ov::Any ov::template_plugin::CompiledModel::get_property(const std::string& name
     } else if (ov::loaded_from_cache == name) {
         return m_loaded_from_cache;
     } else if (ov::execution_devices == name) {
-        return decltype(ov::execution_devices)::value_type{get_plugin()->get_device_name()};
+        return decltype(ov::execution_devices)::value_type{get_plugin()->get_device_name() + "_" +
```

Comment:
we usually separate device_name and device_id using `.`, is not it?

Reply:
Correct